recipes/mosquitto/2.x/conanfile.py

```diff
@@ -47,6 +46,9 @@ def _build_subfolder(self):
     def config_options(self):
         if self.settings.os == "Windows":
             del self.options.fPIC
+        # https://github.com/eclipse/mosquitto/commit/bbe0afbfbe7bb392361de41e275759ee4ef06b1c
+        if tools.Version(self.version) > "2.0.6":
+            self.options["cjson"] = True
```

Comment:
Honestly, I would turn it to True unconditionally, since it seems to be the intent of upstream even before.